MastodonSDK/Sources/MastodonUI/View/Content/UserView.swift

```diff
@@ -38,13 +54,75 @@ public final class UserView: UIView {
     // author username
     public let authorUsernameLabel = MetaLabel(style: .statusUsername)
     
+    public let authorFollowersLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .secondaryLabel
+        return label
+    }()
+    
+    public let authorVerifiedLabel: MetaLabel = {
+        let label = MetaLabel(style: .profileFieldValue)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textAttributes = [
+            .font: UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 15, weight: .regular)),
+            .foregroundColor: UIColor.secondaryLabel
+        ]
+        label.linkAttributes = [
+            .font: UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 15, weight: .semibold)),
+            .foregroundColor: Asset.Colors.brand.color
+        ]
+        label.isUserInteractionEnabled = false
+        return label
+    }()
+    
+    public let authorVerifiedImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.setContentHuggingPriority(.required, for: .horizontal)
+        imageView.setContentCompressionResistancePriority(.defaultHigh, for: .vertical)
+        imageView.setContentHuggingPriority(.required, for: .vertical)
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+    
+    private let verifiedStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        return stackView
+    }()
+    
+    private let verifiedStackCenterSpacerView: UILabel = {
+        let label = UILabel()
+        label.text = " · "
+        label.textColor = .secondaryLabel
+        return label
+    }()
+    
+    private let followButton: UIButton = {
```

Comment:
This was done more or less intentionally as there was no need to expose the distinct type of button, but that said, as it's only used privately anyways it might be smarter to declare the actual type.